paymentsheet-example/src/androidTestDebug/java/com/stripe/android/TestAuthorization.kt

```diff
@@ -51,7 +39,7 @@ class TestAuthorization {
     @Before
     fun before() {
         device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
-        testDriver = PlaygroundTestDriver(device, composeTestRule, screenshotProcessor)
+        testDriver = PlaygroundTestDriver(device, rules.compose, screenshotProcessor)
```

Comment:
I was thinking about delegate (`by ComposeTestRule`) so that we could use `rules` as a `ComposeTestRule`, but it seemed too clever to me. Thoughts?

Reply:
The current implementation seems like the right balance to me.